app/models/account_suggestions.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AccountSuggestions
+  def self.get(account, limit, offset = nil)
+    limit  = limit.to_i
+    offset = offset.to_i
+
+    recommendations = PotentialFriendshipTracker.get(account, limit: limit, offset: offset)
+    recommendations.concat(FollowRecommendation.get(account, limit: limit - recommendations.size)) if recommendations.size < limit && offset.zero?
+    recommendations
```

Comment:
I believe you can still have duplicates between those two sets (someone that is recommended to you both because of your interactions and because they're popular in general), leading to confusing results and a waste of recommendation slots.

Also, I still believe it makes sense to show the end-user the distinction between specially-tailored recommendations and generic ones (especially if the first kind of suggestion can be discarded but not the second).

Reply:
You were just a few minutes early to see a newer version with a v2 endpoint that uses a wrapper object with `source` attribute (`past_interaction` or `global`).

Also the duplicate issue here is now resolved